src/coreclr/jit/gtlist.h

```diff
@@ -126,6 +126,7 @@ GTNODE(UMOD             , GenTreeOp          ,0,GTK_BINOP)
 GTNODE(OR               , GenTreeOp          ,1,(GTK_BINOP|GTK_LOGOP))
 GTNODE(XOR              , GenTreeOp          ,1,(GTK_BINOP|GTK_LOGOP))
 GTNODE(AND              , GenTreeOp          ,1,(GTK_BINOP|GTK_LOGOP))
+GTNODE(AND_NOT          , GenTreeOp          ,0,GTK_BINOP)
```

Comment:
I wonder if a better choice here would be introducing `gtNewSimdAndNotNode` instead of an IR node given that `GT_AND_NOT` only exists as a SIMD operation.

Reply:
FWIW, I have a change stashed that adds `GT_AND_NOT` (for ARM's `bic`), so it may be worth leaving it anyway.

Reply:
Yeah, I left it because I think its generally beneficial here. Its just SIMD today, but there are non-SIMD variants that we can (and probably should) be using as well.

Reply:
Okay, then we can keep it as is.